### ISSUE TITLE
Support for PlayStation

### ISSUE THREAD

**Issue by MwareSolutons** [pavjacko/renative#48]
hi we wonder if you will add support for PlayStation any time soon

**Comment by pavjacko** [comment]
@MwareSolutons support for PS4 is coming in `0.23.0` scheduled within next 2 weeks.
Mind you, you will still need the official PS4 development kit to run it on actual device :)

**Comment by MwareSolutons** [comment]
Hi Pavjacko this is great news i see you are working on 23 at the moment is your planning still to have it implemented in this version ?

**Comment by MwareSolutons** [comment]
Any news on the PS4 integration?

**Comment by pavjacko** [comment]
@MwareSolutons yes. we had bunch of priorities (bug fixes, stabbility, logging) to work on which took some time. almost done with this major work. next big release will aim to broad platform scope.
adopting https://en.wikipedia.org/wiki/Tick%E2%80%93tock_model :)
tick release(s) - widen platform reach
tock releases(s) - stability, coverage, supporting features, bug fixing 

so finishing `tock` release cycle phase in next few days. will be looking into PS4 and few others afterwards

**Comment by MwareSolutons** [comment]
Perfect sounds very good looking forward to it

**Comment by MwareSolutons** [comment]
How about adding support for Linux based devices like set top boxes?

**Comment by SatadruBhattacharjee** [comment]
@MwareSolutons , The linux based set top boxe build systems are too much segmented and licensing are also involved. I think most of them just need a Bundle to be hosted or deployed, which is very similar to the Web Build.

**Comment by MwareSolutons** [comment]
Tried to run the Web variant hosted on a server on a Mag254 device but it just gave black screen

**Comment by SatadruBhattacharjee** [comment]
@MwareSolutons 
To Build
---------

`rnv build -p web`

your deployable web app folder will be located in ./platformBuilds/<APP_ID>_web/public
Just host the content of the public folder.

**Comment by MwareSolutons** [comment]
yes that is what we did but it did not run

**Comment by SatadruBhattacharjee** [comment]
@MwareSolutons , What error it's showing in console ?

**Comment by YazeedAsaad** [comment]
Any updates on the ps4 support @pavjacko ?

**Comment by ali-sao** [comment]
رد علي يزيد يا بافجاكو

**Comment by bruno-sartori** [comment]
Any updates on PS4 support?
